ll-insertions.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const Nd = require('./nd');
+
+module.exports = class LinkedList {
+  constructor() {
+    this.head = null;
+  }
+
+  append(newValue) {
```

Comment:
What if nodes in the list already exist? You need to iterate through the list to insert it at the end.